packages/components/filters/src/filter-menu/header/header.tsx

```diff
@@ -16,26 +21,39 @@ type THeaderProps = {
   /**
    * the function to sort the data
    */
-  onSortRequest?: Function;
+  onSortRequest?: (
+    event: MouseEvent<HTMLButtonElement> | KeyboardEvent<HTMLButtonElement>
+  ) => void;
   /**
    * the width of the menu header
    */
 };
 
 const headerContainerStyles = css`
-  display: flex;
+  color: ${designTokens.colorNeutral40};
+  font-weight: ${designTokens.fontWeight500};
+  font-size: ${designTokens.fontSize20};
+  line-height: ${designTokens.lineHeight20};
+  display: grid;
+  grid-template-columns: 1fr min-content;
```

Comment:
@ddouglasz I updated the header styles so that the menu and the operators input never overflow their available width in the menu